pkgs/games/vinegar/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, wine
+, symlinkJoin
+, makeWrapper
+}:
+
+let
+  version = "1.4.5";
+
+  unwrapped = buildGoModule rec {
+    pname = "vinegar";
+
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "vinegarhq";
+      repo = "vinegar";
+      rev = "v${version}";
+      hash = "sha256-FQ7ldW7cvAhHEcKLP5+IybGsRQdhM21/nTIoTKeU2Gs=";
+    };
+
+    vendorHash = "sha256-eRgmyl3y/E9InYD0q3q1jFhWpLFFBowS8WleeLKGWjM=";
+
+    makeFlags = [
+      "PREFIX=$(out)"
+    ];
+
+    buildPhase = ''
+      runHook preBuild
+      make $makeFlags
+      runHook postBuild
+    '';
+
+    installPhase = ''
+      runHook preInstall
+      make install $makeFlags
+      runHook postInstall
+    '';
+  };
+
+in
+symlinkJoin {
```

Comment:
This should probably just be a `postBuild`.

Reply:
Can you please help a little here.
I am new to nix and this package was my first PR while learning nix.
[Although I have a few PRs in now, I am still learning]

Reply:
@mfrw remove the symlinkJoin and move the unwrapped out of the let and wrap in `postBuild`